physics/rrtmgp_lw_cloud_optics.F90

```diff
@@ -74,55 +72,42 @@ module rrtmgp_lw_cloud_optics
 
 contains
 
-!>\defgroup rrtmgp_lw_cloud_optics_mod GFS RRTMGP-LW Cloud Optics Module
```

Comment:
Same comment about removing Doxygen stuff. We may need to consult Man about this to make sure it doesn't mess up the scientific docs too much.